### PR TITLE
Structure creation

### PR DESCRIPTION
**Pull request by gokul-g-zweck**
constructed a skeleton for project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# full-stack-training
\ No newline at end of file
+# full-stack-training
+Epics
+-RESTful APIs creation
+-Authentication & Autheriztion using Jwt
```
Comment (sharan-zweck) [inline README.md:4]: Fix type

File: service/.gitignore [hunk 2]
```diff
@@ -102,3 +102,5 @@ dist
 
 # TernJS port file
 .tern-port
+#package lock json
```
Comment (sharan-zweck) [inline service/.gitignore:105]: Remove the commented line if unused

File: service/app.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+var createError = require('http-errors');
+var express = require('express');
+var path = require('path');
+var logger = require('morgan');
+
+var indexRouter = require('./routes/index');
+var usersRouter = require('./routes/users');
+
+var app = express();
```
Comment (sharan-zweck) [inline service/app.js:9]: Use let/const as needed

File: service/app.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+// error handler
+app.use(function(err, req, res, next) {
+  // set locals, only providing error in development
+  res.locals.message = err.message;
+  res.locals.error = req.app.get('env') === 'development' ? err : {};
+
+  // render the error page
+  res.status(err.status || 500);
+  res.render('error');
+});
```
Comment (sharan-zweck) [inline service/app.js:32]: Use arrow functions

File: service/bin/www [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+var app = require('../app');
+var debug = require('debug')('service:server');
+var http = require('http');
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+var port = normalizePort(process.env.PORT || '3000');
```
Comment (sharan-zweck) [inline service/bin/www:15]: Use const/let

File: service/bin/www [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    return val;
+  }
+
+  if (port >= 0) {
+    // port number
+    return port;
+  }
+
+  return false;
+}
```
Comment (sharan-zweck) [inline service/bin/www:50]: Use arrow functions

File: service/bin/www [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+      process.exit(1);
+      break;
+    case 'EADDRINUSE':
+      console.error(bind + ' is already in use');
+      process.exit(1);
+      break;
+    default:
+      throw error;
+  }
+}
```
Comment (sharan-zweck) [inline service/bin/www:78]: Same here

File: service/routes/users.js [hunk 8]
```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET users listing. */
+router.get('/', function(req, res, next) {
+  res.send('respond with a resource');
+});
+
+module.exports = router;
```
Comment (sharan-zweck) [inline service/routes/users.js:9]: Use let/const
Use arrow functions

**Review by sharan-zweck — CHANGES_REQUESTED** [review]
Please address the changes

**Review by sharan-zweck — APPROVED** [review]

**Review by sijah-zweck — APPROVED** [review]

**Pull request merged**